test/unit/spec/appWidgets/view/dynamicDropdownViewSpec.js

```diff
@@ -8,83 +8,85 @@ define(['widgets/appWidgets2/view/dynamicDropdownView', 'common/runtime', 'testU
     const DEFAULT_VALUE = 'banana';
     const SELECTOR = 'select.form-control[data-element="input"]';
 
-    beforeEach(function () {
-        this.paramSpec = {
-            data: {
-                defaultValue: DEFAULT_VALUE,
-            },
-        };
-        this.bus = Runtime.make().bus();
-        this.node = document.createElement('div');
-        document.body.appendChild(this.node);
-    });
-
-    afterEach(function () {
-        this.node.remove();
-        window.kbaseRuntime = null;
-    });
-
-    it('should have a valid constructor', function () {
-        const widget = DynamicDropdownView.make({
-            parameterSpec: this.paramSpec,
-            bus: this.bus,
-            initialValue: DEFAULT_VALUE,
+    describe('Dynamic Dropdown View tests', () => {
```

Comment:
These weren't inside a `describe` block, so added one here. The rest is unchanged, save for some `clearRuntime` calls.